### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by kobrikx**
#### What's new:

 - Initial commit

### CODE REVIEW

File: .github/workflows/release-helm-charts.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: Create Helm charts release
+on:
+  push:
+    branches:
+      - develop
```
Comment (DimmKirr) [inline .github/workflows/release-helm-charts.yml:5]: Please make repo's main branch to be develop

File: charts/web-app-ext-cert/templates/istio-gateway.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  selector:
+    # Use Istio default ingress gateway
+    istio: ingressgateway
+  servers:
+    - port:
+        number: 80
+        name: http
+        protocol: HTTP
+      hosts:
+        - {{ .Values.app.name}}.{{ .Values.env}}.{{.Values.istio.domain}}
```
Comment (DimmKirr) [inline charts/web-app-ext-cert/templates/istio-gateway.yaml]: Let's add a brand domain here and in the TLS via another story if we can't do it within the current one.

File: charts/web-app-ext-cert/values.yaml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    imagePullPolicy: Always
+
+  ## Define a list of variables. Use it like <variable-name>:<value>:
+  variables:
+    list:
+      APP_NAME: ""
+
+## Define a list of secrets. Secrets should be pulled to the Parameter store with the path: /env/global/<secret-name>:
+secrets:
+  app: global
```
Comment (DimmKirr) [inline charts/web-app-ext-cert/values.yaml]: Can we we read `/env/squibby/<secret-name>` or only global?
IF we can't, let's add a separate story for this.

File: charts/web-app/templates/deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# Create Deployment (APP deploy)
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.app.name}}-deployment
+  # Use the same namespace we created
```
Comment (DimmKirr) [inline charts/web-app/templates/deployment.yaml]: Created where? 🤔 (don't use `same`, just `Use the namespace we created in this chart`

File: charts/web-app/templates/external-secret.yaml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    name: {{ .Values.app.name}}-secrets
+    kind: SecretStore
+  data:
+    # metadataPolicy to fetch all the tags in JSON format
+    {{ range .Values.secrets.list }}
+    - secretKey: {{ . }}
+      remoteRef:
+        metadataPolicy: Fetch
+        key: "/{{ $.Values.env}}/{{ $.Values.secrets.app}}/{{ . }}"
+{{ end }}
```
Comment (DimmKirr) [inline charts/web-app/templates/external-secret.yaml]: Shouldn't this be indented by 2 levels?

File: charts/web-app-ext-cert/Chart.yaml [hunk 6]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+appVersion: 1.3.0
+description: Web-app for use with already installed Istio or Ingress-nginx
```
Comment (DimmKirr) [inline charts/web-app-ext-cert/Chart.yaml]:
```suggestion
description: Web-app for a cluster with Istio or Ingress-nginx
```

File: charts/web-app-ext-cert/Chart.yaml [hunk 7]
```diff
@@ -0,0 +1,6 @@
...
+name: web-app-without-ingress
```
Comment (DimmKirr) [inline charts/web-app-ext-cert/Chart.yaml]: Doesn't correspond to directory name

Comment (DimmKirr) [reply charts/web-app-ext-cert/Chart.yaml]: Also, it's not clear why is it ext-cert?

**Review by DimmKirr — CHANGES_REQUESTED** [review]
Mostly looks good, have some questions, please address them and feel free to merge.

Please create a story to join two charts into one and add a selector to create istio gateway/nginx ingress or select them as a config.

File: .github/workflows/release-helm-charts.yml (hunk 1 above)
Comment (kobrikx) [reply .github/workflows/release-helm-charts.yml:5]: ok

File: charts/web-app-ext-cert/templates/istio-gateway.yaml (hunk 2 above)
Comment (kobrikx) [reply charts/web-app-ext-cert/templates/istio-gateway.yaml]: Please clarify, what do you want? I don't quite understand.Sorry.

File: charts/web-app-ext-cert/values.yaml (hunk 3 above)
Comment (kobrikx) [reply charts/web-app-ext-cert/values.yaml]: We can, but this chart is intended to be used with the global.

File: charts/web-app/templates/external-secret.yaml (hunk 5 above)
Comment (kobrikx) [reply charts/web-app/templates/external-secret.yaml]: Please clarify, what do you mean? This is tested and working solution.
Thank you.

Comment (DimmKirr) [reply charts/web-app/templates/external-secret.yaml]: From style prospective the closing `{{ end }}` should be indented on the same level as the opening loop. Does it make sense?

**Pull request merged**